app/scripts/modules/core/loadBalancer/loadBalancersTag.component.ts

```diff
@@ -0,0 +1,65 @@
+import {Application} from '../application/application.model.ts';
+
+export class LoadBalancersTagController implements ng.IComponentController {
+
+  public application: Application;
+  public serverGroup: any;
+  public loadBalancers: any[];
+
+  $onInit() {
```

Comment:
missing `public` keyword?  i know it's implied, but ng2 will require you explicitly state the access modifier.  i like this - it feels more java-esque.